apps/dashboard/app/views/projects/_launcher_buttons.html.erb

```diff
@@ -6,53 +6,54 @@
 -%>
 
 <div class="row">
-
   <div class="col">
     <%= button_to(
       submit_project_launcher_path(@project.id, launcher.id),
-      class: 'btn btn-success',
+      class: 'btn btn-success launcher-button',
       title: 'Launch script with cached or default values',
       disabled: disabled,
       params: launcher.quick_launch_params,
       id: "launch_#{launcher.id}"
     ) do
     %>
-      <i class="fa fa-play"></i>
+      Launch
     <%- end %>
   </div>
 
   <div class="col">
     <%= link_to(
       project_launcher_path(@project.id, launcher.id),
-      class: "btn btn-success #{disabled_class}",
+      class: "btn btn-info launcher-button #{disabled_class}",
       id: "show_#{launcher.id}"
       ) do
     %>
-      <i class="fa fa-bars"></i>
+      Options
     <%- end -%>
-  </div>
 
+  </div>
+</div>
+<div class="row">
   <div class="col">
     <%= link_to(
       edit_project_launcher_path(@project.id, launcher.id),
-      class: "btn btn-primary #{disabled_class}",
+      class: "btn btn-primary launcher-button #{disabled_class}",
       id: "edit_#{launcher.id}",
       title: edit_title) do
     %>
-      <i class="fa fa-pencil"></i>
+      Edit
```

Comment:
I believe we have internationalization support for all of these.

Reply:
fixed